crates/nu-protocol/src/ast/cell_path.rs

```diff
@@ -50,10 +53,11 @@ impl PathMember {
         }
     }
 
-    pub fn test_string(val: String, optional: bool) -> Self {
+    pub fn test_string(val: String, optional: bool, insensitive: bool) -> Self {
```

Comment:
I know clippy will only trigger at [three bools](https://rust-lang.github.io/rust-clippy/master/#fn_params_excessive_bools) here. But I still kind of dislike it. Imo it is not clear whether `optional` or `insensitive` comes first naturally, so you can mix up the order easily. This is not a hard veto, but let's think about this.